bindings/c/test/fdb_api.hpp

```diff
@@ -556,6 +559,61 @@ class Database {
 	}
 };
 
+class Tenant final {
+	static const std::string tenantManagementMapPrefix;
+	static const std::string tenantMapPrefix;
+
+public:
+	static TypedFuture<future_var::None> createTenant(Transaction tr, std::string name) {
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_SPECIAL_KEY_SPACE_ENABLE_WRITES, 1);
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_LOCK_AWARE, 1);
+		KeyRef tenantManagementKey = toBytesRef(tenantManagementMapPrefix + name);
+		tr.set(tenantManagementKey, toBytesRef(std::string("")));
+		return tr.commit();
+	}
+
+	static TypedFuture<future_var::None> deleteTenant(Transaction tr, std::string name) {
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_RAW_ACCESS, 1);
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_LOCK_AWARE, 1);
+		KeyRef tenantMapKey = toBytesRef(tenantMapPrefix + name);
```

Comment:
Use the `tenantManagementMapPrefix` here. You probably won't need to interact with `\xff/tenantMap` at all.